examples/energy-management-app/energy-management-common/energy-management-app.matter

```diff
@@ -1328,15 +1328,6 @@ endpoint 1 {
     callback attribute attributeList;
     ram      attribute featureMap default = 1;
     ram      attribute clusterRevision default = 2;
-
-    handle command GetTargetsResponse;
-    handle command Disable;
-    handle command EnableCharging;
-    handle command EnableDischarging;
-    handle command StartDiagnostics;
-    handle command SetTargets;
-    handle command GetTargets;
-    handle command ClearTargets;
```

Comment:
Why are all of these being disabled?  This does not seem to be related to adding callbacks...

Reply:
For some reason a previous commit must have enabled commands in ZAP which meant that the linker was looking for emberAf...Command handlers which aren't required due to the direct handling of commands in the sdk.
Not entirely sure why this wasn't picked up before by other checkers on previous build runs.

Reply:
Alternatively I can leave them enabled and add empty emberAfXyzCommand handlers? Since it is confusing for people who expect to be able to turn on/off commands in ZAP? Either way probably needs to be better documented on the mechanism to instantiate command handling with this cluster.

Reply:
If all the commands are handled entirely in the SDK via CommandHandlerInterface, what you want is:

1. Turn them on in ZAP, so the ZAP config matches reality.  Otherwise you get ZAP complaining about your feature map not matching your enabled command set, etc.
2. Add your cluster to `CommandHandlerInterfaceOnlyClusters` in `src/app/common/templates/config-data.yaml`.

Reply:
ah - didn't know about that!

Reply:
Ok - done this change to config-data.yaml and re-enabled commands in ZAP both in energy-management and all-clusters. Then regen_all.